keyboards/hhkb/ansi/keymaps/shela/keymap.c

```diff
@@ -315,7 +315,7 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                     register_code(r_inner);
                     r_long_pressed = true;
                 }
-                // set_single_persistent_default_layer(_PSEUDO);
+                default_layer_set(1UL << _PSEUDO);
```

Comment:
Why not the `set_single_persistent_default_layer` function?

Reply:
I switch layer frequently, so I pay attention to the lifetime of the EEPROM and try not to make it permanent.

Reply:
Well, the atmega32u4 is rated for 100,000 write/erase cycles, so I'd say you're pretty safe. 

Also, if you're switching a lot, it may make more sense to use the `TG` keycode, instead.

Reply:
Hmmm... `TG` is good idea. I'll look into it, including switching other layers.